common/bucket_utils.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Helper functions for interacting with the buckets."""
+
+from common import logs
+from common import experiment_utils
+
+logger = logs.Logger('bucket_utils')
+
+
+def __using_gsutil():
+    """ Test whether using gsutil format paths. """
+    try:
+        experiment_path_format = experiment_utils.get_cloud_experiment_path()
+    except KeyError:
+        experiment_path_format = 'gs://default'
+
+    return experiment_path_format.startswith('gs://')
+
+
+if __using_gsutil():
+    from common import gsutil as base_utils
+else:
+    # TODO(zhichengcai): local_utils
+    from common import gsutil as base_utils
+
+
+def command(arguments, *args, parallel=False, **kwargs):
+    """ command """
```

Comment:
These docstrings are not sufficient.

Reply:
To pass `make lint` check.

Reply:
The point of `lint` isn't so that you can win a game against a script, it's to remind you to do important things like comment your code properly. Please do that here it shouldn't take long.